test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Builder/TestModels/OrderItem.cs

```diff
@@ -0,0 +1,24 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="OrderLine.cs" company=".NET Foundation">
```

Comment:
update the copyright to refresh the content

Reply:
✅